templates/house_view/btns.html

```diff
@@ -0,0 +1,3 @@
+<div class="btn-group">
+    <a class="btn btn-primary" href="/<str:house_id>/addExpense/" role="button">Add Expense</a>
+</div>
```

Comment:
You should add a new line in the end of the file

Reply:
fixed